README.md

```diff
@@ -1,21 +1,20 @@
-# CodiMD
+![HedgeDoc Logo](https://raw.githubusercontent.com/hedgedoc/hedgedoc-logo/main/LOGOTYPE/SVG/HedgeDoc-Logo%204b.svg)
```

Comment:
The image is a bit too large for a README in my opinion.
Maybe add some size restrictions and center it horizontally.

Reply:
horizontal center is not possible with only markdown

Reply:
That's true. I just checked how nextjs has done that in their README. Seems they just use an image with resolution of 1800x600.

Reply:
I added a resolution tag

Reply:
I replaced the image